docs/zh_cn/tutorials/customize_dataset.md

```diff
@@ -282,175 +319,108 @@ dataset_A_train = dict(
 
 ## 使用 dataset 包装器自定义数据集
 
-MMDetection 也支持非常多的数据集包装器（wrapper）来混合数据集或在训练时修改数据集的分布。
-最近 MMDetection 支持如下三种数据集包装：
+MMEngine 也支持非常多的数据集包装器（wrapper）来混合数据集或在训练时修改数据集的分布，其支持如下三种数据集包装：
 
 - `RepeatDataset`：将整个数据集简单地重复。
 - `ClassBalancedDataset`：以类别均衡的方式重复数据集。
 - `ConcatDataset`：合并数据集。
 
-### 重复数据集（Repeat dataset）
+具体使用方式见 [MMEngine 数据集基类包装](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/basedataset.md#%E6%95%B0%E6%8D%AE%E9%9B%86%E5%9F%BA%E7%B1%BB%E5%8C%85%E8%A3%85)。
```

Comment:
URL is strange

Reply:
Chinese is encoded

Reply:
After the document is generated, it should be replaced uniformly